bin/tx-prover/README.md

```diff
@@ -120,6 +122,28 @@ Then access the Jaeger UI at `http://localhost:16686/`.
 
 If Docker is not an option, Jaeger can also be set up directly on your machine or hosted in the cloud. See the [Jaeger documentation](https://www.jaegertracing.io/docs/) for alternative installation methods.
 
+## Metrics
+
+The proxy includes a service that provides metrics for [Prometheus](https://prometheus.io/docs/introduction/overview/). This service is always enabled and uses the port defined in the configuration file. The metrics are available at the `/metrics` endpoint.
```

Comment:
It's not just the port that's defined in the config file, right? I would also mention config files by name here.